frontend/src/screens/internal-apps/ZapPlanner.tsx

```diff
@@ -87,6 +87,8 @@ export function ZapPlanner() {
   const [amount, setAmount] = React.useState("");
   const [comment, setComment] = React.useState("");
   const [senderName, setSenderName] = React.useState("");
+  const [frequencyValue, setFrequencyValue] = React.useState("1");
```

Comment:
This means by default the payment is done every day. Is that a good default you think? I'd argue monthly payments are probably more common, also that was the default up until now.